src/xunit.performance.api/PerformanceMonitorCounter/BasePerformanceMetric.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Session;
+using Microsoft.Xunit.Performance.Sdk;
+using System.Collections.Generic;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal abstract class BasePerformanceMetric : PerformanceMetric
```

Comment:
Why does `BasePerformanceMetric` inherit from `PerformanceMetric`? Shouldn't it be the other way around? (Or is it just a naming issue?)